### ISSUE TITLE
Suggestion: Export note to gist

### ISSUE THREAD

**Issue by rcrooks** [HappenApps/Quiver#298]
We use Quiver as a team, and one of the things we do frequently is create gists to embed as code samples in docs. It would be great if Quiver could take Github credentials and exports notes directly to public gists.

**Comment by ylian** [comment]
Thanks, Robert.

I assume you are only using gists to host the code snippets, but you actually maintain another website for the documentation. Gists are okay for hosting code snippets, but IMHO are terrible for reading because one note would be mapped into many disjointed gists.

If you are only using gists for hosting code snippet, I wonder what's the benefit of that compared to putting the code snippet in the web page directly.

**Comment by rcrooks** [comment]
Hi Yaogang,

Yes, the docs are hosted elsewhere, we just embed the gists. Two reasons 
for doing that:
1. some bits of code end up in multiple places, and we want to update in 
just one place
2. we use gist-pal, which lets you display lines selectively, highlight 
parts of the code, etc.

Meant to write and tell you version 3 is fantastic!

Yaogang Lian wrote:

**Comment by ylian** [comment]
I see. But how would that work with a Quiver note? If you save the whole snippet in a code cell, export the note as a documentation page and upload the code cell to a public gist, do you have to manually edit the page again to break down the code snippet?

**Comment by rcrooks** [comment]
Right, we'd still have to do that -- unless Quiver got back the embed 
code the way GistPal does. But this is not a high priority enhancement 
for us - we have other ways of doing it.

Yaogang Lian wrote:
